sdk/compute/Microsoft.Azure.Management.Compute/src/Generated/VirtualMachineScaleSetVMsOperationsExtensions.cs

```diff
@@ -1026,9 +1034,13 @@ public static async Task<VirtualMachineScaleSetVM> BeginUpdateAsync(this IVirtua
             /// <param name='instanceId'>
             /// The instance ID of the virtual machine.
             /// </param>
-            public static void BeginDelete(this IVirtualMachineScaleSetVMsOperations operations, string resourceGroupName, string vmScaleSetName, string instanceId)
+            /// <param name='forceDeletion'>
+            /// Optional parameter to force delete a virtual machine from a VM scale set.
+            /// (Feature in Preview)
+            /// </param>
+            public static void BeginDelete(this IVirtualMachineScaleSetVMsOperations operations, string resourceGroupName, string vmScaleSetName, string instanceId, bool? forceDeletion = default(bool?))
```

Comment:
no overload needed

Reply:
@markcowl , I don't understand. This was a generated file. I will ignore this comment.